### ISSUE TITLE
Trained with MNIST dataset

### ISSUE THREAD

**Issue by zjuls** [bamsumit/slayerPytorch#71]
Hello, a lot of thanks for your pytorch code! I have few questions when I tried to train SLAYER with MNIST dataset. 
1. I use the pixel value as the current input so the time window could be less than 20. Does this affect the performance for the SLAYER uses a rate-based loss function?
2. I didn't achieve the expected result with the architecture 28x28-12c5-2a-64c5-2a-10o. Maybe this is because of the neuron model settings. What does each parameter in network.yaml mean? How can I adjust the neuron model configuration? 

Thanks a lot for your reply!
Liam

**Comment by bamsumit** [comment]
20 timesteps should be fine. Refer to this doc for explanation of network.yaml parameters:
https://bamsumit.github.io/slayerPytorch/build/html/slayerParams.html
